draft-ietf-quic-transport.md

```diff
@@ -801,33 +801,41 @@ limit is increased.
 
 ## Flow Credit Increments {#fc-credit}
 
-This document leaves when and how many bytes to advertise in a MAX_STREAM_DATA
-or MAX_DATA frame to implementations, but offers a few considerations.  These
-frames contribute to connection overhead.  Therefore frequently sending frames
-with small changes is undesirable.  At the same time, larger increments to
-limits are necessary to avoid blocking if updates are less frequent, requiring
-larger resource commitments at the receiver.  Thus there is a trade-off between
-resource commitment and overhead when determining how large a limit is
-advertised.
+Implementations decide when and how many bytes to advertise in MAX_STREAM_DATA
+and MAX_DATA frames. This section describes one requirement and offers a few
+considerations.
+
+A receiver MUST NOT wait for a STREAM_DATA_BLOCKED or DATA_BLOCKED frame before
```

Comment:
I understand that we are moving existing text, but "MUST NOT" seems a bit odd to me, as it could be interpreted as a prohibition against the receiver intentionally blocking the stream. Maybe "SHOULD NOT" is better.

Reply:
Yes, it seems unnecessary. Let's do this separately in a design issue though, letting this one be editorial.

Reply:
Ah that's a good point. Rereading the entire paragraph, I think I'm less concerned about readers possibly misinterpreting the text (though, admittedly, I was a bit confused).

Reply:
I've rephrased and moved the text around a bit, so hopefully it's clearer now.

Reply:
Indeed it is. Thank you for the change.